src/ls/completion_item/component.cr

```diff
@@ -20,14 +20,14 @@ module Mint
                 workspace.type_checker.cache[property]?
 
               value =
-                case type.try(&.name)
-                when "String"
+                case type
+                when String
```

Comment:
This would break the functionality since the value of `cache` is not a Crystal type (https://github.com/mint-lang/mint/blob/master/src/type_checkers/types.cr), this was it always goes to the else branch.

Reply:
I got you, thanks for the clarification! ftr, it's actually comparing `TypeChecker::Checkable#name` (whereas `Checkable` is `Type | Record | Variable`), not `type.class#name` as it would with my changes.

Reply:
Yes! Next time I'll be more specific :smile: